cmd/oras/cp.go

```diff
@@ -52,54 +49,88 @@ func copyCmd() *cobra.Command {
 
 ** This command is in preview and under development. **
 
-Example - Copy the artifact tagged with 'v1' from repository 'localhost:5000/net-monitor' to repository 'localhost:5000/net-monitor-copy' 
+Example - Copy the artifact tagged with 'v1' from repository 'localhost:5000/net-monitor' to repository 'localhost:5000/net-monitor-copy':
   oras cp localhost:5000/net-monitor:v1 localhost:5000/net-monitor-copy:v1
 
-Example - Copy the artifact tagged with 'v1' and its referrers from repository 'localhost:5000/net-monitor' to 'localhost:5000/net-monitor-copy'
+Example - Copy the artifact tagged with 'v1' and its referrers from repository 'localhost:5000/net-monitor' to 'localhost:5000/net-monitor-copy':
   oras cp -r localhost:5000/net-monitor:v1 localhost:5000/net-monitor-copy:v1
 
-Example - Copy the artifact tagged with 'v1' from repository 'localhost:5000/net-monitor' to 'localhost:5000/net-monitor-copy' with certain platform
+Example - Copy the artifact tagged with 'v1' from repository 'localhost:5000/net-monitor' to 'localhost:5000/net-monitor-copy' with certain platform:
   oras cp --platform linux/arm/v5 localhost:5000/net-monitor:v1 localhost:5000/net-monitor-copy:v1 
 
-Example - Copy the artifact tagged with 'v1' from repository 'localhost:5000/net-monitor' to 'localhost:5000/net-monitor-copy' with multiple tags
+Example - Copy the artifact tagged with 'v1' from repository 'localhost:5000/net-monitor' to 'localhost:5000/net-monitor-copy' with multiple tags:
   oras cp localhost:5000/net-monitor:v1 localhost:5000/net-monitor-copy:v1,tag2,tag3
 
-Example - Copy the artifact tagged with 'v1' from repository 'localhost:5000/net-monitor' to 'localhost:5000/net-monitor-copy' with multiple tags and concurrency level tuned
+Example - Copy the artifact tagged with 'v1' from repository 'localhost:5000/net-monitor' to 'localhost:5000/net-monitor-copy' with multiple tags and concurrency level tuned:
   oras cp --concurrency 6 localhost:5000/net-monitor:v1 localhost:5000/net-monitor-copy:v1,tag2,tag3
+
+Example - Download an artifact from remote registry to a folder 'local' in OCI image layout:
+  oras cp --to-oci localhost:5000/net-monitor:v1 local:v1
+  oras cp --to-target type=oci localhost:5000/net-monitor:v1 local:v1
```

Comment:
@FeynmanZhou Do we still need this example of the complete flag `--to-target type=oci`?

Reply:
Same applies to other places.

Reply:
The second sample command is not necessary in this example as it has the same effect as the first one (`--to-oci`).

Reply:
@shizhMSFT Do we expect user to use `--to-target type=oci` before adding new options (like new target type, compressed option)? If not, can we remove the long flag and add it when new options comes in?

Reply:
>  remove the long flag and add it when new options comes in

Less is better. Let's do it.

Reply:
removed